source/develop/release-management/features/storage/4k-blocks.md

```diff
@@ -0,0 +1,93 @@
+---
+title: 4k block device support for the storage domains
```

Comment:
This is not good name, since may never support 4k for block storage. Lets call this "4k drive support"

Reply:
Done